server/schedule/checker/joint_state_checker.go

```diff
@@ -37,6 +38,10 @@ func NewJointStateChecker(cluster opt.Cluster) *JointStateChecker {
 // Check verifies a region's role, creating an Operator if need.
 func (c *JointStateChecker) Check(region *core.RegionInfo) *operator.Operator {
 	checkerCounter.WithLabelValues("joint_state_checker", "check").Inc()
+	if c.IsPaused() {
+		checkerCounter.WithLabelValues("joint_state_checker", "skipped").Inc()
```

Comment:
```suggestion
		checkerCounter.WithLabelValues("joint_state_checker", "paused").Inc()
```
it's more intuitive.

Reply:
done.